tests/topotests/bgp_set_aspath_exclude/test_bgp_set_aspath_exclude.py

```diff
@@ -208,7 +206,26 @@ def test_no_bgp_set_aspath_exclude_access_list():
         """
 conf
  route-map r2 permit 6
-  no set as-path exclude as-path-access-list SECOND
+  set as-path exclude 65555
+    """
+    )
+
+    r1.vtysh_cmd(
+        """
+clear bgp *
+    """
+    )
+
+    test_func = functools.partial(bgp_converge, tgen.gears["r1"], expected_3)
+    _, result = topotest.run_and_expect(test_func, None, count=30, wait=0.5)
```

Comment:
I'd give more time here to let it converge under a high load...

Reply:
how much do you want ?